docs/sources/administration/provisioning/_index.md

```diff
@@ -364,8 +364,65 @@ providers:
 
 > **Note:** To provision dashboards to the General folder, store them in the root of your `path`.
 
+## Alerting
+
+You can manage alert objects in Grafana by adding one ore more YAML or JSON
+config files in the [`provisioning/alerting`]({{< relref "../../setup-grafana/configure-grafana/" >}}) directory. Those files will be applied once when starting Grafana. When Grafana
+is running, it's possible to do a hot reload using the [Admin API]({{< relref "../../developers/http_api/admin/#reload-provisioning-configurations" >}}).
+
+### Rules
+
+```yaml
+# config file version
+a1piVersion: 1
+
+# List of rule groups to import or update
+groups:
```

Comment:
See recommendations in sample.yaml for the lines below

Reply:
This is a straight copy of sample.yaml so the user won't find more consulting sample.yaml